<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table7_1</class>
 <widget class="QDialog" name="table7_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Tусi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tmj</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBoxk">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditncp">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tncp</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>80</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Tсп</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tмдр</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEditv">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxm">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Tв</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_td">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tv">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тв</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ктд</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ktd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>λk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>λ0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lambda0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lambdak">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kpp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кпп</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kgp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_R">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кгп</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_F">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_N">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_L">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_knn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gn</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gid">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gид</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Knn</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Kus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_N0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Nu">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kус</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>δ0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kуп</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>δу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kup">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_delta_u">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_delta_0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Kns">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gссд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gсд</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кнс</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gccd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gcd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxn_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>K0i</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kбi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gi</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_q">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
